src/dd-resizable.ts

```diff
@@ -311,23 +317,22 @@ export class DDResizable extends DDBaseImplement implements HTMLElementExtendOpt
 
   /** @internal */
   protected _ui = (): DDUIData => {
-    const containmentEl = this.el.parentElement;
-    const containmentRect = containmentEl.getBoundingClientRect();
+    const { scaleX, scaleY } = Utils.getScaleForElement(this.el);
     const newRect = { // Note: originalRect is a complex object, not a simple Rect, so copy out.
       width: this.originalRect.width,
-      height: this.originalRect.height + this.scrolled,
+      height: (this.originalRect.height + this.scrolled),
       left: this.originalRect.left,
-      top: this.originalRect.top - this.scrolled
+      top: (this.originalRect.top - this.scrolled)
     };
     const rect = this.temporalRect || newRect;
     return {
       position: {
-        left: rect.left - containmentRect.left,
-        top: rect.top - containmentRect.top
+        left: rect.left / scaleX,
```

Comment:
so this has a very different meaning - was relative position, and now it's not (and scaled). Thsi will affect user of those values...

Reply:
I didn't changed the meaning. This is the real position of the element. temporalRect has been updated to be at the right position